tutorials/audio.py

```diff
@@ -0,0 +1,94 @@
+"""
+    mss.tutorials.audio
+    ~~~~~~~~~~~~~~~~~~~
+
+    This python script is meant for generating audio of our choice from text files describing the tutorial.
+
+    This file is part of mss.
+
+    :copyright: Copyright 2021 Hrithik Kumar Verma
+    :copyright: Copyright 2021 by the mss team, see AUTHORS.
+    :license: APACHE-2.0, see LICENSE for details.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+"""
```

Comment:
add a 
`# XYZ don't add credential of API keys here, this has to be done later as secret for a CI call`